pkg/kp/kv.go

```diff
@@ -378,6 +379,39 @@ func (c consulStore) WatchPods(
 	}
 }
 
+// Does the same thing as WatchPods, but does so on all the nodes instead
+func (c consulStore) WatchAllPods(
+	podPrefix PodPrefix,
+	quitChan <-chan struct{},
+	errChan chan<- error,
+	podChan chan<- []ManifestResult,
+) {
+	defer close(podChan)
+
+	kvPairsChan := make(chan api.KVPairs)
+	go consulutil.WatchPrefix(string(podPrefix), c.client.KV(), kvPairsChan, quitChan, errChan, 0)
```

Comment:
you should expose the `pause` (i.e. `0`) parameter to `WatchPrefix` to the caller, our /intent tree is over 40MB which means the fetch might take on the order of 10s of seconds. We can tune consul load by pausing between updates as necessary